### ISSUE TITLE
cannot format xml on CentOS 7

### ISSUE THREAD

**Issue by NewtMa** [redhat-developer/vscode-xml#452]
I was formatting xml on CentOS 7 virtual machine using ssh remote development extension.

It gave this error message:

```
Command 'XML Tools: Format as XML' resulted in an error (Running the contributed command: 'xmlTools.formatAsXml' failed.)
```

XML extension version is 0.16.0;

vscode info is listed below (this is generated on Ubuntu 20 04):
```
Version: 1.55.2
Commit: 3c4e3df9e89829dce27b7b5c24508306b151f30d
Date: 2021-04-13T09:37:02.931Z
Electron: 11.3.0
Chrome: 87.0.4280.141
Node.js: 12.18.3
V8: 8.7.220.31-electron.0
OS: Linux x64 5.4.0-71-generic
```

The command is working fine on my Ubuntu 20 04, but not on CentOS 7.

The XML sample I was formatting has Chinese and English characters both in texts and tags.

Any help would be appreciated.

**Comment by fbricon** [comment]
`xmlTools.formatAsXml` is provided by the competing [XML Tools](https://github.com/DotJoshJohnson/vscode-xml/blob/master/package.json#L57) extension. You should disable it if you want to keep using our vscode-xml extension.

**Comment by NewtMa** [comment]
Oh, my bad, will close this


On 04/14/2021 15:24, Fred Bricon wrote:

Closed #452.
